iota-client/src/api/send.rs

```diff
@@ -101,6 +106,19 @@ impl<'a> SendBuilder<'a> {
         Ok(self)
     }
 
+    /// Set a dust allowance transfer to the builder, address needs to be Bech32 encoded
+    pub fn with_dust_allowance_output(mut self, address: &Bech32Address, amount: u64) -> Result<Self> {
+        if amount < 1_000_000 {
+            return Err(Error::DustError(
+                "Amount for SignatureLockedDustAllowanceOutput needs to be >= 1_000_000".into(),
+            ));
+        }
+        let address = Address::try_from_bech32(&address.to_string())?;
+        let output = SignatureLockedDustAllowanceOutput::new(address, amount).unwrap().into();
```

Comment:
Am thinking that it might be better to transmit the error out by `?` instead of unwarp() which will panic when there is an error.